.gitignore

```diff
@@ -19,3 +19,7 @@ coverage
 
 # bundled files
 lib/bundle.js
+
+package-lock.json
+
+front-end-testing/test/*
```

Comment:
Just realised confusion that can relate to this... my intention was that there was an empty directory called "test" inside of "front-end-testing" but because it was empty it hasn't been pushed up at all. Just made a PR to account for this in the .md file.